maturity.md

```diff
@@ -0,0 +1,80 @@
+| **LC50** | The project clearly defines and documents the copyright ownership of everything that the project produces.                                                                                                                                                                              | **YES** And all source files are with APLv2 header, check by `apache/skywalking-eyes@v0.4.0`. |
+
+**Releases**
+
+| **ID**   | **Description**                                                                                                                                                                                                                                                                 | **Status** |
+| -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------- |
+| **RE10** | Releases consist of source code, distributed using standard and open archive formats that are expected to stay readable in the long term.                                                                                                                                       | **YES** Source release is distributed via [dist.apache.org](https://dist.apache.org/repos/dist/release/incubator/kvrocks/) and linked from [download page](http://kvrocks.incubator.apache.org/download). |
+| **RE20** | The project's PMC (Project Management Committee, see CS10) approves each software release in order to make the release an act of the Foundation.                                                                                                                                                                          | **YES** All releases have been voted at dev@kvrocks and general@incubator, and have at least 3 PPMC's votes. |
+| **RE30** | Releases are signed and/or distributed along with digests that anyone can reliably use to validate the downloaded archives.                                                                                                                                                       | **YES** All releases are signed, and the [KEYS](https://dist.apache.org/repos/dist/release/incubator/kvrocks/KEYS) is available. |
+| **RE40** | The project can distribute convenience binaries alongside source code, but they are not Apache Releases, they are provided with no guarantee. | **YES** User can easily build binaries from source code. And we don't provide binaries as Apache Releases. |
+| **RE50** | The project documents a repeatable release process so that someone new to the project can independently generate the complete set of artifacts required for a release. | **YES** We can follow the [Release guide](https://kvrocks.apache.org/community/create-a-release) to make new Apache Kvrocks release. And so far we had 3 different release managers. |
```

Comment:
```suggestion
| **RE50** | The project documents a repeatable release process so that someone new to the project can independently generate the complete set of artifacts required for a release. | **YES** We can follow the [Release guide](https://kvrocks.apache.org/community/create-a-release) to make new Apache Kvrocks releases, and so far we had 3 different release managers. |
```